### PR TITLE
Ft apple page clone

### PR DESCRIPTION
**Pull request by Trodrige**
This is my work on the apple page clone.

### CODE REVIEW

**Comment by trox115** [comment]
### Status: Changes Required 

Hello @Trodrige , it's very nice to meet you :handshake: . 
Your are doing a very good job with the 4 bottom images and the footer. Those look perfect. But there are some things you need to change. 

**Your header and nav items are not displaying correctly**


You have the stickler files there, but It looks like you have forgotten to activate it on [stickler website](https://stickler-ci.com/). If you have doubts, take a look in this [docs](https://github.com/microverseinc/linters-config/tree/master/css). 

**Checklist**

- [ ] Put header working, with image and navigation.

- [x] Activate your linter on stickler website. 

Do this changes and submit another code review.

**You got this** :muscle: 

If you have any doubt, please reach me on slack. 
**António Fernandes**

**Comment by Trodrige** [comment]
Hello @trox115 

Thank you for the review. I don't understand the issue with the header and the image. This is what I have on my side.

Screen Shot 2019-12-13 at 11 35 05 AM


> 

Please let me know if I'm missing something exactly. I don't understand why the images don't render well on your browser.

**Comment by Trodrige** [comment]
For the meantime, I've added the hover effect on the navbar items.

I'm hoping to hear from you soon.

Regards,
Trodrige

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              <div><img class="subhead" src="./img/your_verse_subhead_small_2x.png" alt="What will be your verse"></div>
+              <div class="hero-video">
+                  <a href="#"></a>
+              </div>
+            </div>
+          <aside>
+              <img src="./img/aside-image-1.jpg" alt="iphone 5s">
+              <img src="./img/aside-image-2.jpg" alt="iphone 5c">
+              <img src="./img/aside-image-3.jpg" alt="Happy birthday">
+              <img src="./img/aside-image-4.jpg" alt="Supplier Responsibility">
```
Comment (suretrust) [inline index.html]: Each of these images should be wrapped in an `<a>` tag because they are links.

**Review by suretrust — APPROVED** [review]
Hi @Trodrige,

### Amazing job! :raised_hands: No deal-breaker, let's get this merged. :+1: 

- [x] I trust you would update your code in accordance with my comment below.


_Good luck with your future projects and happy coding!_ :tada:

**Pull request merged**